src/firewall.py

```diff
@@ -541,6 +591,8 @@ def main(port_v6, port_v4, dnsport_v6, dnsport_v4, method, udp, syslog):
             method = "ipfw"
         elif program_exists('iptables'):
             method = "nat"
+        elif program_exists('pfctl'):
+            method = "pf"
         else:
             raise Fatal("can't find either ipfw or iptables; check your PATH")
```

Comment:
I'm not a not a sshuttle dev/commiter/anything, but shouldn't `pftcl` be part of this message? Like this:

    "can't find either ipfw, pfctl or iptables; check your PATH"